src/SupervisionSuccess/model/produceProjections.js

```diff
@@ -14,13 +14,16 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 // =============================================================================
+import calcBaseline from "./calcBaseline";
+import calcNewOffense from "./calcNewOffense";
 import calcRevocations from "./calcRevocations";
 import calcSavings from "./calcSavings";
 
 /* Number of months to add to the end of chart (chart need to be continued and overflow block),
 This months should not participate in savings and prisonPopulationDiff calculations
 */
 const ADDED_MONTHS = 5;
+const BASE_YEAR = 12;
```

Comment:
Let's add a comment for this new constant, as well.

Reply:
done